libs/model/src/chain-events/ChainEventCreated.command.ts

```diff
@@ -0,0 +1,135 @@
+import {
```

Comment:
This technically looks like a policy reacting to ChainEventCreated events... policies can invoke commands internally.

Reply:
It looks like that but I don't see how I could cleanly map an externally formatted Webhook payload to a policy. Do you mean invoke the policy from the command?

Reply:
Are these events routed to the Outbox? If that's the case, we need an ACL to map external events to internal events. Policies can react to internal events.

ACLs are not part of the domain model, so no need to make them look like commands or policies

Let me know if this is something different that should be considered part of the domain.

Reply:
Yes these events eventually go to the Outbox. In this PR I am not implementing that portion though. I will just be recreating the ACL from `nodeProcessing.ts` in the next PR i.e. use/update `parseEvmEventToContestEvent`.